helper.php

```diff
@@ -2,25 +2,24 @@
 
 // must be run within Dokuwiki
 if (!defined('DOKU_INC')) die();
-require_once(DOKU_INC.'inc/plugin.php');
+require_once(DOKU_INC.'inc/pluginutils.php');
 
 class helper_plugin_schulzevote extends DokuWiki_Plugin {
 
     function __construct() {
         global $ID;
         $data = p_get_metadata($ID, 'schulzevote');
-        $this->prefer = $data['prefer'];
-        $this->candys = $data['candys'];
-        $this->votees = $data['votees'];
+        $this->options = $data['options'];
+        $this->votes = $data['votes'];
```

Comment:
Indeed this is the only part that apparently breaks the backward compatibility.
I will provide a converter for this in a short while, but unfortunately it would have to merge down all votes/voters and disable the "cancel vote" feature for these outdated polls.

Reply:
A converter was introduced to ensure backward-compatibility with former polls, [see here](https://github.com/cosmocode/dokuwiki-plugin-schulzevote/pull/7/files#diff-aa95c45d84019dc4f58e62e37172bc144bbff580540d63fc67612d3f6a635d0cR215-R229).
Naturally, as all votes were previously mangled into the `votees` array these outdated polls are (still) read-only.
Another "bug fix" was introduced to avoid letting the users remove their vote if this latter is closed.